### PR TITLE
Multiple attributes can be the key

### PR DESCRIPTION
**Pull request by joostverhoog**
In Studio Pro 9.19.0, multiple attributes can be the key of a published OData resource (OData version 4 services only)

### CODE REVIEW

File: content/en/docs/refguide/modeling/integration/wrap-services-odata.md [hunk 1]
```diff
@@ -128,23 +124,25 @@ Starting in Studio Pro [9.17](/releasenotes/studio-pro/9.17/), you can select wh
 * **String**
 * **AutoNumber** 
 
-Select an attribute with the following constraints: 
+Select a combination of attribute with the following constraints:
```
Comment (sbont) [inline content/en/docs/refguide/modeling/integration/wrap-services-odata.md]: missing s "attribute**s**"

File: content/en/docs/refguide/modeling/integration/wrap-services-odata.md [hunk 2]
```diff
@@ -128,23 +124,25 @@ Starting in Studio Pro [9.17](/releasenotes/studio-pro/9.17/), you can select wh
...
 ### 5.1 Selecting an Attribute as a Key {#select-key}
 
 To select a different attribute as a key, do the following:
 
 1. Open the **Published OData Resource**. 
 2. In the **Key** section, click **Edit** located next to the **Key** property.
-3. In the **Key Selection** dialog box, move the desired key attribute to the right side and move the old key attribute to the left side. 
-
-Currently, you can only set only a single attribute as a key. 
+3. In the **Key Selection** dialog box, move the key attributes to the right side.
```
Comment (sbont) [inline content/en/docs/refguide/modeling/integration/wrap-services-odata.md:145]: In this line (and in the topics above) we talk about "attributes" (plural), in the previous lines we mention key as a singular thing - should we word it here more consistently?

**Review by sbont — CHANGES_REQUESTED** [review]

File: content/en/docs/refguide/modeling/integration/wrap-services-odata.md (hunk 1 above)
Comment (joostverhoog) [reply content/en/docs/refguide/modeling/integration/wrap-services-odata.md]: That's indeed a typo

File: content/en/docs/refguide/modeling/integration/wrap-services-odata.md (hunk 2 above)
Comment (joostverhoog) [reply content/en/docs/refguide/modeling/integration/wrap-services-odata.md:145]: An entity has a single key (singular), consisting of one or more attributes. Could you indicate where you see inconsistencies?

Comment (sbont) [reply content/en/docs/refguide/modeling/integration/wrap-services-odata.md:145]: I mean that in line 55 of published-odata-resource we mention `3.1 Selecting Attributes as the Key`, in this paragraph title and line 141 we only use the singular of `attribute`. Hope this clarifies

Comment (joostverhoog) [reply content/en/docs/refguide/modeling/integration/wrap-services-odata.md:145]: I see. I changed lines 127, 139 and 141. I think my confusion was that this comment is about line 145, and that line seems fine.

**Review by johannahemminger — APPROVED** [review]

**Pull request merged**